core/includes/form.inc

```diff
@@ -3350,6 +3350,9 @@ function form_process_container($element, &$form_state) {
  * renderable element, to surround it with a <div> and add attributes such as
  * classes or an HTML id.
  *
+ * See the @link https://api.backdropcms.org/form_api#theme_wrappers Form API reference @endlink
```

Comment:
I'm not sure how to use this `@link` and `@endlink` nor how to test if it does what is expected.

@quicksketch  I read this https://api.backdropcms.org/doc-standards#link and took a shot at it.  

Is this correct or any pointers if not?

thanks.